<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="1" failures="0" hostname="DESKTOP-95RCN69" name="pytest" skipped="0" tests="2" time="17.480" timestamp="2022-02-17T14:26:53.137406"><testcase classname="test_dir.test_baidu.TestSearch" name="test_baidu_search_case" time="11.876" /><testcase classname="pytest" name="internal" time="0.000"><error message="internal error">Traceback (most recent call last):
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\_pytest\main.py", line 269, in wrap_session
    session.exitstatus = doit(config, session) or 0
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\_pytest\main.py", line 323, in _main
    config.hook.pytest_runtestloop(session=session)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\_pytest\main.py", line 348, in pytest_runtestloop
    item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 208, in _multicall
    return outcome.get_result()
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\pytest_rerunfailures.py", line 240, in pytest_runtest_protocol
    reports = runtestprotocol(item, nextitem=nextitem, log=False)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\_pytest\runner.py", line 126, in runtestprotocol
    reports.append(call_and_report(item, "call", log))
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\_pytest\runner.py", line 217, in call_and_report
    report: TestReport = hook.pytest_runtest_makereport(item=item, call=call)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\manager.py", line 87, in &lt;lambda&gt;
    firstresult=hook.spec.opts.get("firstresult") if hook.spec else False,
  File "C:\Users\Administrator\AppData\Roaming\Python\Python36\site-packages\pluggy\callers.py", line 203, in _multicall
    gen.send(outcome)
  File "C:\Users\Administrator\Desktop\pyautoTest\conftest.py", line 46, in pytest_runtest_makereport
    capture_screenshots(case_name)
  File "C:\Users\Administrator\Desktop\pyautoTest\conftest.py", line 93, in capture_screenshots
    RunConfig.driver.save_screenshot(image_dir)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 241, in check_response
    raise exception_class(message, screen, stacktrace, alert_text)
selenium.common.exceptions.UnexpectedAlertPresentException: Alert Text: 已经记录下您的使用偏好
Message: unexpected alert open: {Alert text : 已经记录下您的使用偏好}
  (Session info: chrome=98.0.4758.102)</error></testcase></testsuite></testsuites>